lua/paint/blur_cl.lua

```diff
@@ -7,11 +7,11 @@ local paint = paint
 	It doesn't ocupy smalltex1 now. Use it freely)
 ]]
 
-local convarBlur = CreateConVar('paint_blur', 10, FCVAR_ARCHIVE, 'Amount of blur that needs to apply', 0, 100)
-local convarBlurPasses = CreateConVar('paint_blur_passes', 2, FCVAR_ARCHIVE, 'Amount of blur passes that needs to apply', 0, 30)
-local convarBlurFPS = CreateConVar('paint_blur_fps', 20, FCVAR_ARCHIVE, 'How many FPS needed for blur?', 1, 60)
+local convarBlur = CreateConVar('paint_blur', '10', FCVAR_ARCHIVE, 'Amount of blur that needs to apply', 0, 100)
+local convarBlurPasses = CreateConVar('paint_blur_passes', '2', FCVAR_ARCHIVE, 'Amount of blur passes that needs to apply', 0, 30)
+local convarBlurFPS = CreateConVar('paint_blur_fps', '20', FCVAR_ARCHIVE, 'How many FPS needed for blur?', 1, 60)
 
-local texture = GetRenderTargetEx('paint_blur_rt', 256, 256, RT_SIZE_DEFAULT, MATERIAL_RT_DEPTH_NONE, bit.band(2, 256, 32768), 0, IMAGE_FORMAT_RGB888)
```

Comment:
I would just review this line, it may be needed, but I didn't see bit.band accepting more than two arguments in the wiki documentation

Reply:
It can accept varargs.

Reply:
Pretty sure it can break something.

Reply:
> It can accept varargs.

I will replace that back and also update the wiki so it will be known and not throw linter error once the wiki gets scrapped again